scripts/upgrade

```diff
@@ -62,6 +62,34 @@ exec_occ() {
     php$NEXTCLOUD_PHP_VERSION --define apc.enable_cli=1 occ --no-interaction --no-ansi "$@")
 }
 
+#=================================================
+# HANDLE DATABASE MIGRATION FROM MYSL TO PSQL
+#=================================================
+
+mysql_db_pwd=$(ynh_app_setting_get --app=$app --key=mysqlpwd)
+
+if [ -n "$mysql_db_pwd" ]
+then
+    ynh_script_progression --message="Migrating to PostgreSQL database..." --weight=3
+
+    #ynh_backup_if_checksum_is_different --file="$install_dir/config/config.php"
+
+    ynh_psql_test_if_first_run
+    ynh_psql_setup_db --db_user=$db_user --db_name=$db_name #--db_pwd=$(ynh_string_random)
+    #ynh_app_setting_set --app=$app --key=psqlpwd --value=$db_pwd
```

Comment:
Now that we are in packaging v2, the DB should already be provisioned before the upgrade happens so eeeeh